bip-330.mediawiki

```diff
@@ -110,13 +112,13 @@ For announcing and relaying transaction outside of reconciliation, we need an un
 
 Set reconciliation primarily consists of the transmission and decoding of a reconciliation set sketch upon request.
 
-[[File:bip-reconcil/recon_scheme_merged.png|framed|center|Set reconciliation protocol flow]]
+[[File:bip-330/recon_scheme_merged.png|framed|center|Set reconciliation protocol flow]]
```

Comment:
Missing the leading zero

Reply:
oops.

Reply:
Filename too

Reply:
oh god, I should be more attentive, sorry for that.